UPGRADE-2.3.md

```diff
@@ -0,0 +1,12 @@
+UPGRADE FROM 2.3.3 to 2.3.4
```

Comment:
A feature **must not** be published on a patch version but on a **minor** one.

See http://semver.org/.

This PR will be merged for 2.4